tests/peer/test_peerid.py

```diff
@@ -59,7 +53,7 @@ def test_str_more_than_10():
 def test_eq_true():
     random_id_string = ""
     for _ in range(10):
-        random_id_string += random.SystemRandom().choice(ALPHABETS)
+        random_id_string += random.choice(ALPHABETS)
```

Comment:
probably slightly better to use `secrets.choice` here.

Reply:
i don't see the rationale... we just want to sample some characters from the alphabet to build a random string... i don't see the need for "stronger" randomness in this context